src/rust/ide/src/view/text_editor.rs

```diff
@@ -130,11 +130,11 @@ impl TextEditor {
 
         let notifications_sub = self.with_borrowed(|data| data.controller.subscribe());
         executor::global::spawn(process_stream_with_handle(notifications_sub,weak,|notif,this| {
-            this.handle_model_notification(notif)
+            this.handle_controller_notification(notif)
         }));
     }
 
-    fn handle_model_notification(&self, notification:controller::text::Notification)
+    fn handle_controller_notification(&self, notification:controller::text::Notification)
```

Comment:
But this actually handles the model notification, that means "notifications from module's model", doesn't it?

Reply:
yep, accidental change
must have went overboard with the reverse model->controller refactoring